ci/ci.yaml

```diff
@@ -361,25 +363,28 @@ spec:
                 export PIPEKIT_PASSWORD="{{ .Data.data.password }}"
             {{- end }}
       container:
-        image: pipekit13/cli
+        image: pipekitdev/ci-ubuntu
         command:
-          - sh
+          - /bin/bash
           - -c
           - |
-            apk add --no-cache python3 py3-pip
+            apt-get update && apt-get install -y python3 python3-pip
             pip install pipekit-sdk --break-system-packages
 
+            
+            curl -LO https://github.com/pipekit/cli/releases/download/v3.8.2/cli_3.8.2_linux_arm64.deb
```

Comment:
You need to do the thing I suggested to make sure you're always getting the latest. This will be right for about 36 hours :)

Reply:
I was... getting round to that 😂